test/avocado/selenium-machines-storage-pool.py

```diff
@@ -0,0 +1,116 @@
+import time
+import os
+from avocado import skipIf
+from testlib_avocado.machineslib import MachinesLib
+from testlib_avocado.seleniumlib import clickable, invisible
+
+
+
+class MachinesStoragePoolTestSuite(MachinesLib):
+    """
+    :avocado: enable
+    :avocado: tags=machines_s
+    """
+
+    def testCheckStoragePool(self):
+        # sometime element will be not loaded completed, so a short sleep here
+        time.sleep(5)
```

Comment:
As I asked in the previous review hardcoded sleeps should be replaced with wait_* calls.

Reply:
Modify it with a wait_* call.Actually, the reason that i used a sleep here is that sometimes i found the element would not get the number of storage pool in time.It will lead to assertion errors.So the 'sleep' here just want to make sure to get the storage pool number correctly.However, i modify it with 'wait_css('#card-pf-storage-pools')'.I have no idea if there is any better method to do that.Maybe i can learn some knowledge about JavaScript in the future and get the idea with using JS.